chocolatey/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,6 @@
+﻿$app = Get-WmiObject -Class Win32_Product | Where-Object {
```

Comment:
Will this properly remove the virtualenv and everything?

Reply:
It uninstalls lektor and deletes the associated files. I don't think it'll remove virtualenv as its a dependency. But I'll look into it!